server/sql_info_fetcher.go

```diff
@@ -215,7 +217,7 @@ func (sh *sqlInfoFetcher) zipInfoForSQL(w http.ResponseWriter, r *http.Request)
 		case <-timer.C:
 			cancelFunc()
 		}
-		err = <-errChan
+		err = dumpCPUProfile(errChan, &buf, zw)
```

Comment:
should we  add `wg.wait` here for `go sh.catchCPUProfile(reqCtx, pprofTime, &buf, errChan)` to finish, making sure `&buf` has been correctly writed